tool/tsh/common/tsh.go

```diff
@@ -354,8 +353,16 @@ type CLIConf struct {
 	// X11ForwardingTimeout can optionally set to set a timeout for untrusted X11 forwarding.
 	X11ForwardingTimeout time.Duration
 
-	// Debug sends debug logs to stdout.
+	// Debug sets log level to debug and sends logs to stdout.
 	Debug bool
+	// DebugSet specifies whether the flag was set by the user.
+	DebugSet bool
```

Comment:
Taken out of context, I'd probably guess `DebugSet` to mean what `Debug` actually means: do we want debug output or not. I think the "set by user" part wouldn't cross my mind.

Perhaps we can use more verbose naming like `DebugSetByUser`?

Reply:
Yeah, it's not the best name, I just copied what was used in other places where we use that kingpin method. I'll change it.